.github/workflows/bootstrap.yml

```diff
@@ -67,13 +67,17 @@ jobs:
         with:
           python-version: "3.12"
       - name: Bootstrap clingo
+        env:
+          SETUP_SCRIPT_EXT: ${{ matrix.runner == 'windows-latest' && 'ps1' || 'sh' }}
+          SETUP_SCRIPT_SOURCE: ${{ matrix.runner == 'windows-latest' && './' || 'source ' }}
+          USER_SCOPE_PARENT_DIR: ${{ matrix.runner == 'windows-latest' && '$env:userprofile' || '$HOME' }}
```

Comment:
Apparently the windows job doesn't error on failure, but it reports a :heavy_check_mark: CI. I was expecting a failure here: https://github.com/spack/spack/actions/runs/9911941054/job/27385832628 and indeed clingo is not building (but CI is :heavy_check_mark: )

Reply:
Ugh right, powershell doesn't exit properly on native executable (non powershell commandlet) exit codes. I'll add the fix and tag you ASAP.